modules/secure_storage_connector/outputs.tf

```diff
@@ -0,0 +1,11 @@
+output "storage_account_name" {
+  value = module.storage.account.name
+}
+
+output "storage_container_name" {
+  value = module.storage.container.name
+}
+
+output "managed_identity_client_id" {
+  value = azurerm_user_assigned_identity.default.client_id
+}
```

Comment:
maybe we should also output the tenant ID, because customers will need to know if when they go to create the team in the UI (then again, they can also look it up in the azure portal if they don't know it)

Reply:
Agree, may as well output it. It can't hurt, may be useful later.